doc/releases/changelog-dev.md

```diff
@@ -39,10 +39,23 @@
   >>> print(energy)
   -1.1258709813834058
   ```
-  
-* New PennyLane-inspired `sketch` and `sketch_dark` styles are now available for drawing circuit diagram graphics. 
+
+* New PennyLane-inspired `sketch` and `sketch_dark` styles are now available for drawing circuit diagram graphics.
   [(#2709)](https://github.com/PennyLaneAI/pennylane/pull/2709)
 
+* Added new device abstract base class `QutritDevice` for simulation of devices with wires of three dimensions. Users can currently make measurements with `qml.state()` and `qml.probs()`.
+* Added operation `qml.QutritUnitary` for applying user-specified unitaries to `QutritDevice`.
+
+  ```pycon
+  >>> dev = qml.device("default.qutrit", wires=1)
+  >>> U = np.array([[1, 1, 0], [1, -1, 0], [0, 0, np.sqrt(2)]]) / np.sqrt(2)
+  >>> with qml.tape.QuantumTape() as tape:
+  ...     qml.QutritUnitary(U, wires=0)
+  ...     return qml.probs(wires=0)
```

Comment:
```suggestion
  ...     qml.probs(wires=0)
```
I don't think `return` is needed for tapes since this is a context and not a function. The example will also need to be updated to take into account the new PR structure.